src/pages/about.js

```diff
@@ -1,85 +1,101 @@
-            frameBorder="0"
-            scrolling="0"
-            width="158px"
-            height="30px"
-          >
-            &nbsp;
-          </iframe>
-        </p>
-        <p>
-          Victory is a set of modular charting components for React and React
-          Native. Victory makes it easy to get started without sacrificing
-          flexibility. Create one of a kind data visualizations with fully
-          customizable styles and behaviors. Victory uses the same API for web
-          and React Native applications for easy cross-platform charting.
-        </p>
-        <p>
-          Victory is helmed by Formidable’s{" "}
-          <a href="https://github.com/boygirl"> Lauren Eastridge</a>.
-        </p>
-        <p>
-          <a href="https://github.com/FormidableLabs/victory/graphs/contributors">
-            See Victory Contributors
-          </a>
-        </p>
+const Divider = styled.hr`
+  border: none;
+  border-bottom: 1px solid ${({ theme }) => theme.color.red};
+  margin-top: 2rem;
+`;
 
-        <h2 id="showcase">Victory in Use</h2>
+const Section = styled.div`
+  margin-bottom: ${({ theme }) => theme.spacing.lg};
+`;
+
+const Subheader = styled.h2`
+  font-size: 3rem;
+`;
+
+const About = ({ location }) => {
+  return (
+    <Page location={location}>
+      <h1>Victory: Charting for React and React Native</h1>
+      <p>
+        {/*
+         * TODO: Customize these buttons
+         * https://github.com/FormidableLabs/formidable-landers/issues/175
+         */}
+        <iframe
+          src="https://ghbtns.com/github-btn.html?user=formidablelabs&repo=victory&type=star&count=true&size=large"
+          frameBorder="0"
+          scrolling="0"
+          width="160px"
+          height="30px"
+        >
+          &nbsp;
+        </iframe>
+        <iframe
+          src="https://ghbtns.com/github-btn.html?user=formidablelabs&repo=victory&type=watch&count=true&size=large&v=2"
+          frameBorder="0"
+          scrolling="0"
+          width="160px"
+          height="30px"
+        >
+          &nbsp;
+        </iframe>
+        <iframe
+          src="https://ghbtns.com/github-btn.html?user=formidablelabs&repo=victory&type=fork&count=true&size=large"
+          frameBorder="0"
+          scrolling="0"
+          width="158px"
+          height="30px"
+        >
+          &nbsp;
+        </iframe>
+      </p>
+      <p>
+        Victory is a set of modular charting components for React and React
+        Native. Victory makes it easy to get started without sacrificing
+        flexibility. Create one of a kind data visualizations with fully
+        customizable styles and behaviors. Victory uses the same API for web and
+        React Native applications for easy cross-platform charting.
+      </p>
+      <p>
+        Victory is helmed by Formidable’s{" "}
+        <a href="https://github.com/boygirl"> Lauren Eastridge</a>.
```

Comment:
Is this a double-space with the `{" "}` and the space before Lauren between the ending a tag?